roles/openstack-stack/templates/heat_stack.yaml.j2

```diff
@@ -142,18 +142,17 @@ resources:
 #            cluster_id: {{ stack_name }}
 #      public_key: {{ ssh_public_key }}
 
-{% if openstack_flat_secgrp|bool %}
-  flat-secgrp:
+  basic-secgrp:
```

Comment:
Can we call this `common-secgrp` instead? I was really confused at first, trying to figure out what a basic security group would mean.

If I understand it correctly, this contains rules that are common to all nodes in the cluster, right?